bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -50,6 +55,8 @@ async def autoreview_enabled(self) -> bool:
 
     async def refresh_cache(self) -> bool:
         """Updates TalentPool users cache."""
+        # Wait until logged in to ensure bot api client exists
+        await self.bot.on_guild_available()
```

Comment:
What is the difference between this and `wait_until_guild_available`?

Reply:
The difference is that this is the wrong one :) `on_guild_available` is the event that sets the available flag. Nice catch :D

Reply:
https://github.com/python-discord/bot/pull/1813/commits/4e61411d2be105ad1e68fa6211d3c95fc3d92b3b